WordPress/Sniffs/WP/DeprecatedFunctionsSniff.php

```diff
@@ -334,7 +349,7 @@ class WordPress_Sniffs_WP_DeprecatedFunctionsSniff extends WordPress_AbstractFun
 			'version' => '2.8.0',
 		),
 		'get_the_author_description' => array(
-			'alt'     => 'get_the_author_meta( \'description\' )',
+			'alt'     => 'get_the_author_meta(\'description\')',
```

Comment:
I realise this is only going to be displayed as a message to the debugging developer but is there any rationale for preferring no-spaces over spaces? Is this how WP core defines the alternatives?